addons/website_sale_delivery/models/sale_order.py

```diff
@@ -14,19 +14,22 @@ class SaleOrder(models.Model):
         compute='_compute_amount_delivery',
         string='Delivery Amount',
         help="The amount without tax.", store=True, tracking=True)
+    # VFE TODO this field help should be updated, bc the amount can be with tax depending
+    # on the current user groups.
 
     def _compute_website_order_line(self):
         super(SaleOrder, self)._compute_website_order_line()
         for order in self:
             order.website_order_line = order.website_order_line.filtered(lambda l: not l.is_delivery)
 
-    @api.depends('order_line.price_unit', 'order_line.tax_id', 'order_line.discount', 'order_line.product_uom_qty')
+    @api.depends('order_line.price_subtotal')
```

Comment:
I know it's the same compute method, so if one changes so will the other; but i'd prefer to explicitely have `price_total` in the deps as well